core/src/main/java/com/graphhopper/routing/util/DataFlagEncoder.java

```diff
@@ -156,6 +164,18 @@ public int defineWayBits(int index, int shift) {
         carBwdMaxspeedEncoder = new EncodedDoubleValue("car bwd maxspeed", shift, speedBits, speedFactor, 0, maxPossibleSpeed, true);
         shift += carBwdMaxspeedEncoder.getBits();
 
+        /* Value range: [3.0m, 5.4m] */
```

Comment:
Wow, I think this was the fastest fix I've ever seen on OSM, one problem is already fixed: http://www.openstreetmap.org/note/857064